lib/ui/compositing.dart

```diff
@@ -545,30 +597,32 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// for more details.
   // Values above must match constants in //engine/src/sky/compositor/performance_overlay_layer.h
   void addPerformanceOverlay(int enabledOptions, Rect bounds) {
-    _addPerformanceOverlay(enabledOptions,
-                           bounds.left,
-                           bounds.right,
-                           bounds.top,
-                           bounds.bottom);
+    _addPerformanceOverlay(enabledOptions, bounds.left, bounds.right, bounds.top, bounds.bottom);
   }
-  void _addPerformanceOverlay(int enabledOptions,
-                              double left,
-                              double right,
-                              double top,
-                              double bottom) native 'SceneBuilder_addPerformanceOverlay';
+
+  void _addPerformanceOverlay(
+    int enabledOptions,
+    double left,
+    double right,
+    double top,
+    double bottom,
+  ) native 'SceneBuilder_addPerformanceOverlay';
 
   /// Adds a [Picture] to the scene.
   ///
   /// The picture is rasterized at the given offset.
-  void addPicture(Offset offset, Picture picture, { bool isComplexHint = false, bool willChangeHint = false }) {
-    int hints = 0;
-    if (isComplexHint)
-      hints |= 1;
-    if (willChangeHint)
-      hints |= 2;
+  void addPicture(
+    Offset offset,
+    Picture picture, {
+    bool isComplexHint = false,
+    bool willChangeHint = false,
+  }) {
+    final int hints = (isComplexHint ? 1 : 0) | (willChangeHint ? 2 : 0);
```

Comment:
@liyuqian this was, debatably, the best compromise I could find to make both dartfmt and the analyzer happy without expanding this code section even more with {} braces and such. Details in the PR comments.

Reply:
LGTM